addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -44,6 +44,23 @@ def default_get(self, fields):
     message = fields.Html('Message')
     send_mail = fields.Boolean('Send Email', default=True, help="If checked, the partners will receive an email warning they have been added in the document's followers.")
 
+    def _prepare_subscribe_message_values(self, wizard, document, model_name, email_from):
```

Comment:
Would give additional_msg_values as optional additional values, more extensible that giving "email_from" (would keep document and model_name). We could just do
```
message_values = {
    ...
}
if additional_msg_values:
    message_values.update(**additional_msg_values)
```